src/user/test/mocks/user.service.mock.ts

```diff
@@ -34,7 +34,12 @@ export class UserServiceMock implements Required<UserService> {
     _name: string,
     _password: string,
     _role: Role,
-  ): Promise<boolean> {
-    return true;
+  ): Promise<User> {
+    return {
+      id: 1,
+      name: 'testName',
+      password: 'testPassword',
```

Comment:
If you want you can use [faker library](https://www.npmjs.com/package/@faker-js/faker) to generate fake data